about.html

```diff
@@ -0,0 +1,127 @@
+            and get a chance to apply them to real life projects.
+             
+        </p>
+        <p class="please-contact">Please contact us as per Email for any further questions about PyLadies Workshop 2023</p>
+        <a href="mailto:workshop2023@pyladies.org" target="_blank">workshop2023@pyladies.org</a>
+    </Section>
+
+    <!-- Logo info Section-->
+    <section class="logo-info">
+        <h2 class="logo-title">PyLadies Namibia 2023 Logo</h2>
+        <hr>
+        <p class="logo-desc">The logo of PyLadies Namibia Workshop 2023 was decided through the logo competition frm 20 May to 21 May</p>
+        <a href="index.html"><img src="images/logo.jpg" alt="Logo"></a>    
+    </section>
+
+    <!-- Past info Section -->
+    <section class="past-info">
+        <h2 class="past-info-title">See the past PyLadies Namibia Workshops</h2>
+        <hr>
+        <p class="past-info-desc">Take a look at the last two PyLadies Namibia Workshops which took place in Oshakati and in Walvis Bay</p>
+        <div class="last-images">
+            <div class="last-image-wrapper wrapper-1">
+                <img src="images/past-1.jpg" alt="Oshakati PyLadies Namibia"/>
+                <div class="last-info info-1">
+                    <h3 class="last-year">2018</h3>
+                    <p class="last-venue">PyLadies Namibia in Oshakati.</p>
+                </div>
+            </div>
+            <div class="last-image-wrapper wrapper-2">
+                <img src="images/past-2.jpg" alt="Walvis Bay PyLadies Namibia"/>
+                <div class="last-info">
+                    <h3 class="last-year">2020</h3>
+                    <p class="last-venue">PyLadies Namibia in Walvis Bay.</p>
+                </div>
+            </div>
+        </div>
+        </div>
+    </section>
+
+     <!-- Partners Section -->
+    <section class="partners">
+        <h2 class="partner-title">Partner</h2>
+        <hr>
+        <div class="partners-list">
+            <a rel="noopener noreferrer" target="_blank" href="https://www.mozilla.org/en-US/"><img src="images/mozilla.svg" alt="Mozilla"></a>
+            <a rel="noopener noreferrer" target="_blank" href="https://www.google.com/en-US/"><img src="images/google.svg" alt="Google"></a>
+            <a rel="noopener noreferrer" target="_blank" href="https://www.naver.com/en-US/"><img src="images/naver.svg" alt="Naver"></a>
+            <a rel="noopener noreferrer" target="_blank" href="https://www.daumkakao.com/en-US/"><img src="images/daumkakao.svg" alt="Daumkakao"></a>
+            <a rel="noopener noreferrer" target="_blank" href="https://www.airbnb.com/en-US/"><img src="images/airbnb.svg" alt="Airbnb"></a>    
+        </div>
+    </section>
+
+    <!-- Footer Section -->
+    <section class="footer">
+        <a href="index.html"><img src="images/logo.jpg" alt="pyladies-logo"></a>
+        <p class="footer-desc">2022 PyLadies Namibia <br> Same right reserved</p>
+    </section>
```

Comment:
Kindly replace the `section` tag here with `footer` tag since this part is the footer of webpage 🤗